### ISSUE TITLE
error when combining use and query/options

### ISSUE THREAD

**Issue by sokra** [webpack/webpack#3556]
**Do you want to request a *feature* or report a *bug*?**
usability bug

**What is the current behavior?**
`{ use: "babel-loader", options: {} }` is valid but doesn't work

**What is the expected behavior?**
`{ use: "babel-loader", options: {} }` should be an error.

**Comment by bebraw** [comment]
Instead of `{ use: "babel-loader", options: {} }`, `{ loader: "babel-loader", options: {} }` is expected, right? Why not to allow former format, though? If you want to use `use`, then you would have to expand to

```javascript
{
  use: [
    {
      loader: 'babel-loader',
      options: {}
    }
  ]
}
```

**Comment by TheLarkInn** [comment]
Do you want me to bake this into https://github.com/webpack/webpack/pull/3557?

**Comment by TheLarkInn** [comment]
if use is string && options is passed as sibling property, error?

**Comment by TheLarkInn** [comment]
Fixed with #3563
